downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.6.0 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201.6.0/
+active: 2201.6.0
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201.6.0
+    - /downloads/swan-lake-release-notes/2201.6.0-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.6.0-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.6.0
+
+<em>2201.6.0 (Swan Lake Update 6) is the sixth update release of Ballerina Swan Lake, and it includes a new set of features and significant improvements to the compiler, runtime, standard library, and developer tooling. It is based on the 2022R4 version of the Language Specification.</em> 
+
+## Update Ballerina
+
+Update your currrent Ballerina installation directly to 2201.6.0 by using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist pull 2201.6.0` to update to this latest distribution.
```

Comment:
@keizer619 Since we are not changing the sematics of `bal dist update` we should be able to use `bal dist update` here?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6975#discussion_r1209005775.